### ISSUE TITLE
Propose README

### ISSUE THREAD

**Issue by CarlosNacher** [ZFTurbo/segmentation_models_3D#18]
Hi ZFTurbo,

It is very important to realize that backbones expects inputs between 0, 255. So I think that is a good idea to specify it in the `README.md` file, for avoid unexpected behaviour.

I noticed this because if I didn't use Batch Normalization and I used RESNET, when my data was in the range 0, 1, as RESNET subtracts the average (which is around 120), all the data stayed at -120.something and the network didn't learn anything. If you use batchnormalization this is solved because it re-normalizes the batch, but if you don't use it the nets won't train.

**Comment by Nexer8** [comment]
@CarlosNacher, I was looking for a training example, but this repo lacks that. Could you provide me with one?
